examples/gno.land/r/gnoland/home/home.gno

```diff
@@ -143,6 +144,15 @@ func socialLinks() ui.Element {
 	}
 }
 
+func playgroundSection() ui.Element {
+	return ui.Element{
+		ui.H3("Gno Playground, a DevX tool to write and deploy smart contracts directly on Gno.land"),
```

Comment:
@alexiscolin can we update it as "Build, Deploy, and Share Smart Contracts with Playground" and resolve this comment?